blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,103 @@
+
+In earlier Julia versions, stack traces were printed without much structure.
+Everything was printed in bold red which made it a bit difficult to find the information one was looking for.
+As an example, here is a screenshot from an error thrown on 0.5:
+
+![Stacktrace in 0.5](/images/blog/2017-02-24-repl0.6/0.5stacktrace.png)
+
+Finding the relevant information can be quite difficult, especially the file name and line numbers since they are not aligned in any way.
+In 0.6, this is what is currently shown for the same error:
+
+![Stacktrace in 0.6](/images/blog/2017-02-24-repl0.6/0.6stacktrace.png)
+
+As can be seen, stack trace presentation has improved significantly: Stack traces no longer print in red, while associated error messages continue to print in red.
+Stack traces also no longer print entirely in bold; only function names and line information now appear in bold.
+These changes visually separate the most-frequently sought information (the associated error message, and function names and line information) from the surrounding text, making that information much easier to pick out.
+Additionally, each stack frame in the stacktrace prints numbered, making individual stack frames easier to refer to.
+(Apart from these changes to stacktrace presentation, BoundsErrors no longer display with the associated collection's contents, decreasing visual noise.)
+"Furthermore, you can now customize the colors of the stack frame functions and line information by setting the environment variables `JULIA_STACKFRAME_FUNCTION_COLOR` and `JULIA_STACKFRAME_LINEINFO_COLOR` respectively.
+For example, setting
+
+```julia
+julia> ENV["JULIA_STACKFRAME_LINEINFO_COLOR"] = :cyan;
+
+julia> ENV["JULIA_STACKFRAME_FUNCTION_COLOR"] = :yellow;
+```
+
+and rerunning the example above yields
+
+![Stacktrace in 0.6 with configured colors](/images/blog/2017-02-24-repl0.6/0.6stacktrace_conf.png)
+
+In addition to the visual changes of the stack trace, there is also a new interactive feature, possible now that we can refer to stack frames by number.
+After a stack trace has been printed, you can simply enter the number of one of the stack frames in the REPL, press `CTRL + Q` and the file will be opened in
+the editor at the file and line number relevant to that stack frame.
```

Comment:
"the editor" -> "an editor"
"at the line number" (mentioning the file again is redundant)
"relevant" -> "corresponding"